app-arch/libdeflate/libdeflate-1.19.ebuild

```diff
@@ -18,14 +18,31 @@ fi
 
 LICENSE="MIT"
 SLOT="0"
-IUSE="+gzip static-libs +utils +zlib test"
-RESTRICT="!test? ( test )"
+# the zlib USE-flag enables support for zlib
+# the test USE-flag programs depend on sys-libs/zlib for comparison tests
+IUSE="+gzip static static-libs +utils +zlib test"
```

Comment:
We shouldn't add (I'm saying add because this is a pretty new pkg) USE=static-libs unless we really really need it, see https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302, so just drop it if you can.

But also, why have we got USE=static? Does libdeflate install any executables?

Reply:
With `USE=utils` it builds and installs `/usr/bin/libdeflate-gzip` and `/usr/bin/libdeflate-gunzip`. If `static-libs` and `static` are set those will be linked statically. I'll drop it.